hs_core/models.py

```diff
@@ -2007,6 +2007,47 @@ def update(cls, element_id, **kwargs):
 
         super(FundingAgency, cls).update(element_id, **kwargs)
 
+    @classmethod
+    def migrate_from_crossref_ror(cls, apps, schema_editor):
+        """Migrate from CrossRef to ROR"""
```

Comment:
if it would be helpful, I can get you a db dump with the exsiting FundingAgencies so that you can test this migration?